pkg/services/pluginsintegration/clientmiddleware/oauthtoken_middleware.go

```diff
@@ -69,19 +64,19 @@ func (m *OAuthTokenMiddleware) applyToken(ctx context.Context, pCtx backend.Plug
 
 			switch t := req.(type) {
 			case *backend.QueryDataRequest:
-				t.Headers[tokenHeaderName] = authorizationHeader
+				t.Headers[backend.OAuthIdentityTokenHeaderName] = authorizationHeader
 				if idTokenHeader != "" {
-					t.Headers[idTokenHeaderName] = idTokenHeader
+					t.Headers[backend.OAuthIdentityIDTokenHeaderName] = idTokenHeader
```

Comment:
`backend.OAuthIdentityIDTokenHeaderName` seems to be `X-Id-Token`

https://github.com/grafana/grafana-plugin-sdk-go/blob/c5aa822be58253fd954e921aec00241833d9ca4c/backend/http_headers.go#L17

but the `idTokenHeaderName` from the removed const was `X-ID-Token`

(capital "D" vs lowercase "d")

Are we sure this isn't going to cause any issues?

Reply:
I don't _believe_ so, but I will hold off merging this before confirming myself. There's no rush on this and can also probably wait for the expert in this area @marefr!

Reply:
Had a quick look into this. I think all is fine

```go
const OAuthIdentityIDTokenHeaderName = "X-Id-Token"

func getHTTPHeadersFromStringMap(headers map[string]string) http.Header {
	httpHeaders := http.Header{}

	for k, v := range headers {
         // when k == "X-ID-Token" 
         // then textproto.CanonicalMIMEHeaderKey(k) == "X-Id-Token"
		if textproto.CanonicalMIMEHeaderKey(k) == OAuthIdentityIDTokenHeaderName {
			httpHeaders.Set(k, v)
		}
//...
```

Reply:
We're already referring to them in https://grafana.com/developers/plugin-tools/how-to-guides/data-source-plugins/add-authentication-for-data-source-plugins#extract-a-header-from-an-http-request so should be fine I think